src/machines/jetson_machine/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM qutms_driverless:ros-pytorch
+# qutms_driverless_base generated by qutms_driverless/docker/Makefile
+
+ARG USERNAME
+ARG HOST_UID
+# dialout group is used for compatability with mac
+ARG HOST_GROUP=dialout
+
+# copy in perception files and install YOLOv5 vision dependancies
+COPY --chown=${HOST_UID}:${HOST_GROUP} ./src/perception /home/${USERNAME}/driverless_ws/src/perception
```

Comment:
instead of copying the whole directory, set the workdir to `/tmp`, copy `jetson_requirements.txt` only and then run the pip install using a short path (`pip3 install -r jetson_requirements.txt`).

See the pattern we use for `conda_requirements` in the roscube machine dockerfile